config.yaml.sample

```diff
@@ -188,9 +188,16 @@ toi-store:
 
 # Configuration for the tiles of interest prune/garden command.
 toi-prune:
+  # location of tileserver logs
+  tile-traffic-log-path: ../tileserver/nohup.out
+
   # Connection and query configuration for a RedShift database containing
   # request information for tiles.
   redshift:
+    # if database-uri is not specified, postgresql configuration above is used
```

Comment:
@iandees great suggestion. So maybe in the context of it's fields we can rename redshift to `traffic-history` or just `history`?